### PR TITLE
[feature] Add LaWAM policy support

### PR DESCRIPTION
**Pull request by Nemo-1024**
## Summary

This PR adds an in-tree `lawam` policy adapter for LaWAM so it can be used through the standard LeRobot SFT and evaluation interfaces.

Main changes:
- registers `LaWAMConfig` / `LaWAMPolicy` in the policy factory
- ports the LaWAM policy backend, latent-world batching/runtime, flow-matching action head, and LAM components under `src/lerobot/policies/lawam/`
- adds train/eval preprocessing for LeRobot image, state, action, and task batches
- supports original LaWAM `.pt` checkpoints plus separate LAM checkpoint/YAML loading
- documents install, SFT, eval, release checkpoint dimensions, and local training decoder notes
- adds focused unit tests for registration, batch conversion, padded action dimensions, action queue behavior, and flow-horizon masking

## Notes

Released LaWAM checkpoints use a 32-dimensional padded flow action/state space internally. The adapter slices model actions back to the LeRobot environment action dimension for inference. During training, the collator keeps the configured action window while masking flow loss to `floor(policy.flow_horizon_sec * policy.action_hz)` valid action tokens.

## Validation

Conda environment: Python 3.12 with PyTorch 2.7.1+cu128, CUDA available.

Commands run:
- `ruff check src/lerobot/policies/lawam tests/policies/lawam/test_lawam.py`
- `python -m py_compile $(find src/lerobot/policies/lawam -type f -name '*.py' | sort) tests/policies/lawam/test_lawam.py`
- `python -m pytest tests/policies/lawam/test_lawam.py -q`
- `python -m pip check`
- `lerobot-eval` smoke test with LaWAM LIBERO SFT release checkpoint, LIBERO env, 1 episode / 1 step
- `lerobot-train` smoke test from LaWAM pretrain checkpoint on a local LeRobot-format LIBERO batch, 1 optimizer step

The eval and train smoke tests both ran through the LeRobot CLI in the conda environment.

### CODE REVIEW

**Comment by Nemo-1024** [comment]
Closing this draft to reopen with conventional PR naming.
